server/app.ts

```diff
@@ -41,7 +41,7 @@ export default function createApp(services: Services): express.Application {
   app.use(authorisationMiddleware())
   app.use(setUpCsrf())
 
-  app.get(
+  app.use(
```

Comment:
fix here is to call the page components not just on `get`, but on `post` etc. too